pkg/v1/providers/infrastructure-vsphere/v0.7.10/ytt/overlay-windows.yaml

```diff
@@ -0,0 +1,515 @@
+          New-KubeProxyServiceInterface
+          & C:\k\antrea\Install-OVS.ps1 -ImportCertificate $false -LocalFile C:\k\antrea\ovs-win64.zip
+          Set-NetFirewallProfile -Profile Domain,Public,Private -Enabled False
+
+          # Setup Services
+          $nssm = (Get-Command nssm).Source
+          & $nssm set kubelet start SERVICE_AUTO_START
+          & $nssm install kube-proxy "C:\k\kube-proxy.exe" "--proxy-mode=userspace --kubeconfig=$KubeProxyConfig --log-dir=C:\var\log\kube-proxy --logtostderr=false --alsologtostderr"
+          & $nssm install antrea-agent "C:\k\antrea\bin\antrea-agent.exe" "--config=C:\k\antrea\etc\antrea-agent.conf --logtostderr=false --log_dir=C:\var\log\antrea --alsologtostderr --log_file_max_size=100 --log_file_max_num=4"
+          & $nssm set antrea-agent DependOnService kube-proxy ovs-vswitchd
+          & $nssm set antrea-agent Start SERVICE_AUTO_START
+
+          # Start Services
+          start-service kubelet
+          start-service kube-proxy
+          start-service antrea-agent
+      postKubeadmCommands:
+      - powershell C:/Temp/antrea.ps1 -ExecutionPolicy Bypass
+      users:
+      #@overlay/match by=overlay.index(0)
+      #@overlay/replace
+      - name: capv
+        groups: Administrators
+        sshAuthorizedKeys:
+        - #@ data.values.VSPHERE_SSH_AUTHORIZED_KEY
+        sudo: ALL=(ALL) NOPASSWD:ALL
+
+#@overlay/match by=overlay.subset({"kind":"MachineDeployment", "metadata":{"name": "${ CLUSTER_NAME }-md-0"}})
+---
+apiVersion: cluster.x-k8s.io/v1alpha3
+kind: MachineDeployment
+metadata:
+  labels:
+    cluster.x-k8s.io/cluster-name: #@ data.values.CLUSTER_NAME
+  name: #@ "{}-md-0-windows-containerd".format(data.values.CLUSTER_NAME)
+spec:
+  clusterName: #@ data.values.CLUSTER_NAME
+  replicas: #@ data.values.WORKER_MACHINE_COUNT
+  selector:
+    matchLabels:
+      cluster.x-k8s.io/cluster-name: #@ data.values.CLUSTER_NAME
+  template:
+    metadata:
+      labels:
+        cluster.x-k8s.io/cluster-name: #@ data.values.CLUSTER_NAME
+        node-pool: #@ "{}-worker-pool".format(data.values.CLUSTER_NAME)
+    spec:
+      bootstrap:
+        configRef:
+          name: #@ "{}-md-0-windows-containerd".format(data.values.CLUSTER_NAME)
+      clusterName: #@ data.values.CLUSTER_NAME
+      infrastructureRef:
+        name: #@ "{}-windows-containerd".format(data.values.CLUSTER_NAME)
+      version: #@ data.values.KUBERNETES_VERSION
+#@overlay/match by=overlay.subset({"kind": "Secret", "metadata":{"name": "${ CLUSTER_NAME }"}})
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: #@ data.values.CLUSTER_NAME
+stringData:
+  username: #@ data.values.VSPHERE_USERNAME
+  password: #@ data.values.VSPHERE_PASSWORD
+#@ end
+---
+apiVersion: addons.cluster.x-k8s.io/v1alpha3
+kind: ClusterResourceSet
+metadata:
+  annotations:
+    tkg.tanzu.vmware.com/addon-type: metadata/tkg
```

Comment:
This CRS should not include this annotation as it will get used for metadata updates during the cluster upgrade process.